docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -73,17 +74,72 @@ the light client isn’t expired.
 
 After having successfully verified a misbehaviour, the endpoint will execute the jailing and slashing of the malicious validators similarly as in the evidence module. 
 
-### Current limitations:
 
-- This only handles light client attacks, not double signing. In the future, we will add the code to also verify double signing.
+### Double Signing Attack
+
+A double signing attack, also known as an equivocation,
+occurs when a validator sends two different votes for a block in the same round of a consensus instance. 
+The CometBFT consensus operates with multiple rounds of voting at each block height 
```

Comment:
Could add more information here. E.g., two votes of the same type in the same round for a block at the same height, etc.

Reply:
changed in `396b18b3`